packages/flutter_tools/lib/src/android/android_sdk.dart

```diff
@@ -37,7 +37,7 @@ final RegExp _numberedAndroidPlatformRe = RegExp(r'^android-([0-9]+)$');
 final RegExp _sdkVersionRe = RegExp(r'^ro.build.version.sdk=([0-9]+)$');
 
 /// The minimum Android SDK version we support.
-const int minimumAndroidSdkVersion = 25;
+const int minimumAndroidSdkVersion = 29;
```

Comment:
We shouldn't track this in two places.  Can you update the `create.dart` usage (which populates the `.iml` template) to `kAndroidSdkMinVersion` and delete this?

Also, I wonder why the ide templates aren't using that, they hardcode the SDK version instead of using `{{androidSdkVersion}}`:
https://github.com/flutter/flutter/search?q=%22Android+API+25+Platform%22&unscoped_q=%22Android+API+25+Platform%22

Can you update those manually to 29?  I created https://github.com/flutter/flutter/issues/63522 to change these to use the template context.

Reply:
sure. Done. 

re: the ide template stuff, ya it's a bit unclean. I think it was a template to unpack IntelliJ projects for the Flutter SDK itself rather than a Flutter project so it doesn't have the same templates. Still worth cleaning up.